drivers/timer/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ zephyr_sources_if_kconfig(              loapic_timer.c)
 zephyr_sources_if_kconfig(              apic_timer.c)
 zephyr_sources_ifdef(CONFIG_ALTERA_AVALON_TIMER altera_avalon_timer_hal.c)
 zephyr_sources_if_kconfig(              nrf_rtc_timer.c)
+zephyr_sources_if_kconfig(              stm32_lptim_timer.c)
```

Comment:
Update commit title, should be something like:
`drivers/timer: stm32: Add ....`